bin/health-check.rb

```diff
@@ -4,14 +4,27 @@
 
 require 'conjur_client'
 
+conjur_api = ConjurClient.api
+
 begin
-  conjur_api = ConjurClient.api
   if ConjurClient.version == 5
     conjur_api.resources limit: 5
   else
     conjur_api.resource("#{ConjurClient.account}:user:admin").exists?
   end
+  
   puts "Successfully validated Conjur credentials."
 rescue
-  raise "Error: There is an issue with your Conjur configuration. Please verify that the credentials are correct and try again."
+  raise "Error: There is an issue with your Conjur configuration. Please verify" \
+        " that the credentials are correct and try again."
+end
+
+if ConjurClient.version == 4
+  hf_id = ConjurClient.v4_host_factory_id
+  
+  if !conjur_api.resource(URI::encode(hf_id, "/")).exists?
+    raise "Error: There is an issue with your Conjur configuration. Please" \
+          " verify that your Conjur policy contains a host factory named " \
+          "'#{hf_id.split(':')[-1]}' under the '#{ConjurClient.policy}' policy."
```

Comment:
this would show the HF name as something like `pcf/pcf-apps` if the policy was `pcf`, but it should just show `pcf-apps`

Reply:
Fixed

```
root@cee6b32939da:/app# ./bin/health-check.rb
Successfully validated Conjur credentials.
./bin/health-check.rb:26:in `<main>': Error: There is an issue with your Conjur configuration. Please verify that your Conjur policy contains a host factory named 'pcf-apps' under the 'pcf' policy. (RuntimeError)
```